### ISSUE TITLE
ideas

### ISSUE THREAD

**Issue by starsx2016** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#670]
Hi!

any plan for a DARK theme mode for the interface?

thanks for all your hard work

Matt

**Hardware (please complete the following information):**
 - Pump model, for example 54123
 - 4 or 6 wires
 - PCB type
 - Level shifter type,

**Software (please complete the following information):**
 - Version

**Your message**

**Comment by visualapproach** [comment]
Yes, it's pulled to the dev branch from @colinpeddle

**Comment by colinpeddle** [comment]
Yeah it’s a very rudimentary implementation but it does work for saving your eyes :-). I’ve been testing a correction for the occasional flashing as the page loads (especially if the connection is slow or your device is slow), along with several other visual advancements for the app, which hopefully I’ll be able to do a pull for soon!
